bin/plugin/commands/common.js

```diff
@@ -33,6 +45,8 @@ exports.getModuleData = async ( modulesDir ) => {
 			const moduleData = {
 				slug: path.basename( moduleDir ),
 				focus: path.basename( path.dirname( moduleDir ) ),
+				// Set default values, in order to allow to avoid errors when ordering, if set property would be override.
+				experimental: false,
```

Comment:
Why is this needed? Every module needs to have an `Experimental` header, so I think we should remove this default and instead ensure that it is required below, when calling `Array.filter` to remove incomplete entries.

Reply:
Mainly due this property is set dynamic via the headers, in case a header is not set we ensure this property is initialized to a `false` value.

Reply:
@mitogh See my reply to the other comment. I like your idea there with a warning, let's rely primarily on validation like that. In other words, I still think the default here can be removed as this will be validated later anyway.